CommandLine/testing/Tests/draw_tiles_test.cpp

```diff
@@ -0,0 +1,23 @@
+#include "TestHarness.hpp"
+
+#include <gtest/gtest.h>
+
+TEST(Regression, draw_tiles_test) {
+  if (!TestHarness::windowing_supported()) return;
+  auto test_harness = LAUNCH_HARNESS_FOR_SOG(TestConfig());
+  if (!test_harness) FAIL() << "Game could not be run.";
+
+  test_harness->wait();  // Let the game render a frame first.
+  ASSERT_TRUE(test_harness->game_is_running())
+      << "Game stopped running unexpectedly";
+
+  test_harness->screen_save("./test-harness-out/enigma_draw_tiles_test.png");
+
+  test_harness->close_window();
+  bool game_running = test_harness->game_is_running();
+  for (int i = 0; game_running && i < 10; ++i) {

```

Comment:
This is starting to be common code; we may just want to put it in the test harness (as an optional method to call, if not as part of the default dtor).